test/Microsoft.AspNetCore.Sockets.Tests/HttpConnectionDispatcherTests.cs

```diff
@@ -101,6 +104,37 @@ public class HttpConnectionDispatcherTests
             }
         }
 
+        [Theory]
+        [InlineData(TransportType.WebSockets, "/ws", 404)]
+        [InlineData(TransportType.ServerSentEvents, "/sse", 404)]
+        public async Task EndPointThatOnlySupportsLongPollingRejectsOtherTransports(TransportType transportType, string path, int status)
```

Comment:
Would it be a little cleaner to instead provide the supported transport type and the provided type in the test params? This way you don't have to have separate tests for each of the supported transports. You would have to keep the `EndPointThatOnlySupportsWebSocketsAndSSERejectsLongPolling` since you removed the streaming enum though.